local/php/php_builtin_server.go

```diff
@@ -61,9 +60,5 @@ require $script;
 `)
 
 func (p *Server) phpRouterFile() string {
-	path := filepath.Join(p.homeDir, fmt.Sprintf("php/%s-router.php", name(p.projectDir)))
-	if _, err := os.Stat(filepath.Dir(path)); os.IsNotExist(err) {
-		_ = os.MkdirAll(filepath.Dir(path), 0755)
-	}
-	return path
+	return filepath.Join(p.tempDir, fmt.Sprintf("%s-router.php", p.Version.Version))
```

Comment:
For review, I moved the routing file to the same location as the FPM config.
This way I can always create and clean the directory no matter the SAPI